README.md

```diff
@@ -619,6 +621,7 @@ Table of Contents
   * [canister.io](https://canister.io/) — 20 free private repositories for developers, 30 free private repositories for teams to build and store Docker images
   * [Whales](https://github.com/Gueils/whales) — A tool to automatically dockerize your applications for free.
   * [PWD](https://labs.play-with-docker.com/) — Play with Docker. A simple, interactive and fun playground to learn Docker
+  * [Gitlab](https://gitlab.com) - Per-repo container registry.  10GB limit.
```

Comment:
prefer we list them just once

Reply:
I get that, but they offer quite a lot of free services.  Which section would they all best fit under?

Reply:
maybe a larger re-org for developer tooling is needed and we move a bunch of them in there?